src/Lunapark/ActionF.purs

```diff
@@ -32,49 +33,49 @@ derive instance functorTouchF ∷ Functor TouchF
 
 _lunaparkActions = SProxy ∷ SProxy "lunaparkActions"
 type LUNAPARK_ACTIONS = R.FProxy ActionF
-type WithAction r = R.Run (lunaparkActions ∷ LUNAPARK_ACTIONS|r) Unit
+type LunaparkActionsEffect r = ( lunaparkActions ∷ LUNAPARK_ACTIONS | r )
```

Comment:
I don't mind, but could you please share your motivation here? Is it because `BaseEffect` and`WithAction` is unclear?

Reply:
`BaseEffect`? You prob mean `BaseRun`

the reason is that it's easier to understand when Run is not in `type`

- do one thing and do it well
- it's better to not use type aliases when can (I have learned this from https://github.com/purescript-contrib/purescript-argonaut-codecs/pull/74#discussion_r431510193 `I think F in Foreign is a similar case, and a mistake also, as again the underlying types are pretty simple without the synonym`)

Reply:
This change means that instead of this 
```purescript 
myAwesomeTest :: forall r. Lunapark r
myAwesomeTest = do 
  go "http://whatever.com"
  clickEl selector
```
End-users will need to write this 
```purescript
myAwesomeTest:: forall r. Run (LunaparkBaseEffects + LunaparkEffect + LunaparkActionsEffect + r)
myAwesomeTest = do 
  go "http://whatever.com"
  clickEl selector
```

Removing `Lunapark r` complicates basic use-case and propagates implementation details to the call site, I think it should be preserved. I'm fine with removing `WithAction`, `BaseRun` and using `Run (Whatever r)` inside the library, but don't agree with removing main library monad synonim from public API. I'd say that making `Lunapark r` a newtype might be good idea, but not in this pr 🙂 

Personally, I think that this `Lunapark r` (not `WithAction r` and definitely not `BaseRun r`) are somewhat like `CodecJson`, not `F` (ugh... `F` from foreign is terrible)

Reply:
1. returned `type Lunapark`

2. re newtyping - not sure if I like this idea

I'm fine with doing

```
Run (LunaparkBaseEffects + LunaparkEffect + LunaparkActionsEffect + r)
```

actually I removed BaseRun so I could easiter to this to see what is going on

I like when I see `ok, I'm using the Run lib, not something else`